hazelcast/src/test/java/com/hazelcast/jet/impl/submitjob/clientside/upload/JobUploadClientSuccessTest.java

```diff
@@ -175,23 +179,33 @@ public void test_stress_jarUpload_whenResourceUploadIsEnabled() {
 
         ExecutorService executorService = Executors.newFixedThreadPool(10);
 
+        // Shared client for ExecutorService threads
+        HazelcastInstance client = createHazelcastClient();
+        JetClientInstanceImpl jetService = (JetClientInstanceImpl) client.getJet();
+
+        ConcurrentSkipListSet<String> submittedJobNames = new ConcurrentSkipListSet<>();
```

Comment:
it's called on local thread only, no need for concurrent structure

Reply:
Job names are added by the executorService in different threads